inc/admin/upgrader.php

```diff
@@ -40,6 +40,15 @@ function rocket_upgrader() {
 
 	$page = filter_input( INPUT_GET, 'page', FILTER_SANITIZE_STRING );
 
+	if (
+		did_action( 'wp_rocket_upgrade' )
+		&&
+		'wprocket' === $page
```

Comment:
very minor change request,
I think reversing the order of the conditions will be better as this method is called with `admin_init` so in most cases the `$page` will not equal `wprocket` so we don't need to do the second check `did_action`

```suggestion
		'wprocket' === $page
		&&
		did_action( 'wp_rocket_upgrade' )
```

also, I was thinking that here:
https://github.com/wp-media/wp-rocket/blob/7eb62835e5ffa2b3c79c7625fc7921fbe57739d3/inc/admin/upgrader.php#L21
we do this action so why not defining variable here instead of using `did_action`, I think this will work also.

This is very minor so I will approve it.